liquidator/index.js

```diff
@@ -118,7 +122,7 @@ async function run(logger, address, pollingDelay, priceFeedConfig, liquidatorCon
       });
     }
 
-    while (true) {
+    for (;;) {
```

Comment:
OOC: why change this?

Reply:
I'm following the `eslint:recommended`: [disallows constant expressions in conditions](https://eslint.org/docs/rules/no-constant-condition).